models/Recipe.js

```diff
@@ -1,9 +1,33 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title: String,
+  level: {
+    type: String,
+    ENUM: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"],
+    default: "Easy Peasy"
+  },
+  ingredients: Array,
+  cuisine: String,
+  dishType: {
+    type: String,
+    enum: ["Breakfast", "Dish", " Snack", " Drink", "Dessert", "Other"]
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg"
+  },
+  duration: {
+    type: Number,
+    min: 0
+  },
+  creator: String,
+  created: {
+    type: Date,
+    default: Date.now()
+  }
```

Comment:
Be the more precise you can when creating your schema, so you are sure about the relevance of the information in your DB